changelog/changes/2022-02-22-enable-kerberos-for-nfsv4.md

```diff
@@ -0,0 +1,6 @@
+- Bring in dependencies for NFS4 with Kerberos both in kernel and userspace. Tested against NFS4.1. [PR#1664](https://github.com/flatcar-linux/coreos-overlay/pull/1664)
```

Comment:
It's a bit long now for the release notes. Since we link to this PR here which also has the same info, I suggest to shorten it to one bullet point

Reply:
Makes sense, removed in latest commit.